static/app/components/events/interfaces/spans/index.tsx

```diff
@@ -58,29 +61,74 @@ class SpansInterface extends Component<Props, State> {
 
   renderTraceErrorsAlert({
     isLoading,
-    numOfErrors,
+    errors,
+    parsedTrace,
   }: {
     isLoading: boolean;
-    numOfErrors: number;
+    errors: TraceError[] | undefined;
+    parsedTrace: ParsedTraceType;
   }) {
     if (isLoading) {
       return null;
     }
 
-    if (numOfErrors === 0) {
+    if (!errors || errors.length <= 0) {
       return null;
     }
 
     const label = tn(
       'There is an error event associated with this transaction event.',
       `There are %s error events associated with this transaction event.`,
-      numOfErrors
+      errors.length
     );
 
+    const errorsMap: {
+      [spanId: string]: {operation: string; errorsCount: number};
+    } = {};
+
+    errors.forEach(error => {
+      if (!errorsMap[error.span]) {
+        if (parsedTrace.rootSpanID === error.span) {
+          errorsMap[error.span] = {
+            operation: parsedTrace.op,
```

Comment:
Maybe add a comment here why `parsedTrace.op` is used here (I'm guessing it doesn't show up in parsedTrace.spans), since it's a little confusing at first glance that `error.span` is an id, not a span.

Reply:
Comments added in 5cdb684